samples/simulate-ui-login.ps1

```diff
@@ -0,0 +1,143 @@
+﻿[CmdletBinding()]
+Param(
+    [Parameter(Mandatory=$true)]
+    [string]$Appliance,
+    [Parameter(Mandatory=$true)]
+    [string]$IdentityProvider,
+    [Parameter(Mandatory=$true)]
+    [string]$UserName,
+    [Parameter(Mandatory=$true)]
+    [SecureString]$Password,
+    [Parameter(Mandatory=$false)]
+    [Switch]$Insecure = $false,
+    [Parameter(Mandatory=$true)]
+    [string]$AssetName,
+    [Parameter(Mandatory=$true)]
+    [string]$AccountName
+)
+
+if($Insecure)
+{
+    # GET notification/v3/Status
+    echo "GET status"
+    $status = Get-SafeguardStatus -Appliance $Appliance -Insecure

```

Comment:
You can use the `$Insecure` variable like this:
```PowerShell
Get-SafeguardStatus -Appliance $Appliance -Insecure:$Insecure
```

That should allow you to shorten this.